analyzer-core/src/main/java/ffb/analyzer/core/web/webclient/SimpleWebClient.java

```diff
@@ -1,28 +1,43 @@
 package ffb.analyzer.core.web.webclient;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.CollectionType;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
 import org.apache.http.client.methods.CloseableHttpResponse;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.client.methods.HttpPost;
 import org.apache.http.client.methods.HttpRequestBase;
 import org.apache.http.impl.client.HttpClients;
 
-import java.io.IOException;
-import java.util.List;
-
+/**
+ *  Web client with basic functionality to send HTTP requests.
+ */
 public class SimpleWebClient implements BaseWebClient {
 
+    private static final List<DeserializationFeature> DEFAULT_DESERIALIZATION_OPTIONS = List.of(
```

Comment:
Created #13 - Expose Deserialization Options in SimpleWebClient. This should be more configurable than it is currently.